definitions/mobile.d.ts

```diff
@@ -205,12 +205,11 @@ declare module Mobile {
 		 * Filters data for specified platform.
 		 * @param {string} platform The platform for which is the device log.
 		 * @param {string} data The input data for filtering.
-		 * @param {string} projectDir The project's root directory.
 		 * @param {string} pid @optional The application PID for this device.
 		 * @param {string} logLevel @optional The logging level based on which input data will be filtered.
 		 * @return {string} The filtered result based on the input or null when the input data shouldn't be shown.
 		 */
-		filterData(platform: string, data: string, projectDir: string, pid?: string, logLevel?: string): string;
```

Comment:
This is required for NativeScript CLI:
https://github.com/NativeScript/nativescript-cli/blob/master/lib/services/ios-log-filter.ts#L11